src/pocket_book.py

```diff
@@ -8,12 +7,11 @@
 from enum import Enum
 
 from PyPDF2 import PdfFileReader, PdfFileWriter
-from PySimpleGUI import theme, Button, Text, Input, InputText, FilesBrowse, Window, WIN_CLOSED, Image as guiImage
 
-import src.pdfbooklet_new as pdfbooklet_new
+import pdfbooklet_new as pdfbooklet_new
```

Comment:
it seems like this fails the CI, maybe move you father folder in flask one folder backwords